issue concerning dc-chain docker image

Hello, 
currently the Dockerfile for the dc-chain is missing a `\` at a command line 39 : 
	
> && make all toolchain_profile=$dc_chain makejobs=$makejobs use_custom_dependencies=1
> 	&& rm -rf /opt/toolchains/dc/kos

Also, I tried to build an image , I got this error using the toolchain_profile=`14.1.1-dev`  ( same issue with 13.2.1 at least):

> gcc/build -I../../gcc-14.1.1/gcc/../include  -I../../gcc-14.1.1/gcc/../libcpp/include  \
>         -o build/genversion.o ../../gcc-14.1.1/gcc/genversion.cc
> g++ -c   -g -O2     -DIN_GCC -DCROSS_DIRECTORY_STRUCTURE   -fno-exceptions -fno-rtti -fasynchronous-unwind-tables -W -Wall -Wno-narrowing -Wwrite-strings -Wcast-qual -Wmissing-format-attribute -Wconditionally-supported -Woverloaded-virtual -pedantic -Wno-long-long -Wno-variadic-macros -Wno-overlength-strings  -Wno-error -DHAVE_CONFIG_H  -DGENERATOR_FILE -I. -Ibuild -I../../gcc-14.1.1/gcc -I../../gcc-14.1.1/gcc/build -I../../gcc-14.1.1/gcc/../include  -I../../gcc-14.1.1/gcc/../libcpp/include  \
>         -o build/gengtype-lex.o gengtype-lex.cc
> cc1plus: fatal error: gengtype-lex.cc: No such file or directory
> compilation terminated.
> make[3]: *** [Makefile:2994: build/gengtype-lex.o] Error 1
> make[3]: *** Waiting for unfinished jobs....
> rm gfdl.pod gcc.pod gcov-dump.pod gcov-tool.pod fsf-funding.pod gpl.pod cpp.pod gcov.pod lto-dump.pod
> make[3]: Leaving directory '/opt/toolchains/dc/kos/utils/dc-chain/build-gcc-sh-elf-14.1.1-pass1/gcc'
> make[2]: *** [Makefile:4685: all-gcc] Error 2
> make[2]: Leaving directory '/opt/toolchains/dc/kos/utils/dc-chain/build-gcc-sh-elf-14.1.1-pass1'
> make[1]: *** [Makefile:1047: all] Error 2
> make[1]: Leaving directory '/opt/toolchains/dc/kos/utils/dc-chain/build-gcc-sh-elf-14.1.1-pass1'
> make: *** [scripts/gcc-pass1.mk:13: build-sh4-gcc-pass1] Error 1

--- a/utils/dc-chain/docker/Dockerfile
+++ b/utils/dc-chain/docker/Dockerfile
@@ -14,20 +14,23 @@ MAINTAINER KallistiOS <cadcdev-kallistios@lists.sourceforge.net>
 
 # Installing prerequisites
 RUN apk --update add --no-cache \
-	build-base \
-	patch \
-	bash \
-	texinfo \
-	libjpeg-turbo-dev \
-	libpng-dev \
-	curl \
-	wget \
-	git \
-	python3 \
-	subversion \
-	gmp-dev \
-	elfutils-dev \
-	&& rm -rf /var/cache/apk/*
+        build-base \
+        gmp-dev \
+        mpfr-dev \
+        mpc1-dev \
+        patch \
+        bash \
+        texinfo \
+        flex \
+        curl \
+        wget \
+        git \
+        subversion \
+        elfutils-dev \
+        libjpeg-turbo-dev \
+        libpng-dev \
+        python3 \
+        && rm -rf /var/cache/apk/*
 
 # Making Sega Dreamcast toolchains
 # You may adapt the KallistiOS repository URL if needed
@@ -36,7 +39,7 @@ ARG makejobs=2
 RUN mkdir -p /opt/toolchains/dc \
 	&& git clone https://git.code.sf.net/p/cadcdev/kallistios /opt/toolchains/dc/kos || git clone https://github.com/KallistiOS/KallistiOS.git /opt/toolchains/dc/kos \
 	&& cd /opt/toolchains/dc/kos/utils/dc-chain \
-	&& make all toolchain_profile=$dc_chain makejobs=$makejobs use_custom_dependencies=1
+	&& make build toolchain_profile=$dc_chain makejobs=$makejobs \
 	&& rm -rf /opt/toolchains/dc/kos
 
 # Stage 2

